cmd/finch/nerdctl_windows.go

```diff
@@ -197,7 +247,7 @@ func handleBindMounts(systemDeps NerdctlCommandSystemDeps, nerdctlCmdArgs []stri
 }
 
 // handles --output/-o for build command.
-func handleOutputOption(systemDeps NerdctlCommandSystemDeps, nerdctlCmdArgs []string, index int) error {
+func handleOutputOption(systemDeps NerdctlCommandSystemDeps, _ *config.Finch, nerdctlCmdArgs []string, index int) error {
```

Comment:
nit: any particular reason to prefer pointers, Nothing wrong with it just want to know the reasoning.

Reply:
If I'm reading the GitHub highlighting correctly, I think that you commenting on the `*config.Finch` pointer.

Since the config was originally defined within the `fc` as such, I replicated it rather than trying to convert it into a new type. Additionally, I thought if there were other future opportunities for the Finch Config to change handler behavior, then the entire object would already be available.